bucket/tableplus.json

```diff
@@ -22,10 +32,10 @@
         ]
     ],
     "checkver": {
-        "url": "https://tableplus.com/blog/2018/09/changelogs-windows.html",
-        "regex": "Build (\\d+)</h3>"
+        "url": "https://tableplus.com/win/version.xml?sign=&version=x64",
+        "regex": "sparkle:version=\"(?<build>\\d+)\"\\s+sparkle:shortVersionString=\"([\\d.]+)\""
     },
     "autoupdate": {
-        "url": "https://s3.amazonaws.com/tableplus-win-builds/$version/TablePlusSetup.exe"
+        "url": "https://s3.amazonaws.com/tableplus-win-builds/$matchBuild/TablePlusSetup.exe"
```

Comment:
Why match? Add replace to checkver with `${1}.${build}` and then use appropriate version substitution it should be $patchVersion I guess